backend/src/utils/migrations.py

```
@@ -10,6 +40,16 @@ def run_migrations():
         alembic_cfg = Config("alembic.ini")
         command.upgrade(alembic_cfg, "head")
         logger.info("Database migrations completed successfully")
+    except CommandError as e:
+        # Handle case where database has a stale revision that no longer exists
+        if "Can't locate revision" in str(e):
+            logger.warning(
+                f"Database has stale revision, clearing and stamping: {str(e)}"
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)